clients/drcachesim/simulator/caching_device_stats.cpp

```diff
@@ -176,4 +178,9 @@ caching_device_stats_t::reset()
     num_hits = 0;
     num_misses = 0;
     num_child_hits = 0;
+    num_incl_invalidates = 0;
```

Comment:
Hmm, how about `num_inclusive_invalidates` to be clear

Reply:
Done.